### DISCUSSION TITLE
How to set up Lighthouse integration?

### DISCUSSION THREAD

**Discussion by jakst** [vercel/vercel#4485] [Help] [answered]
I've added the Lighthouse integration to my Vercel account, but I cannot get it to work it all. Is it supposed to enrich PR's with lighthouse data? Because I don't get any of that.

In the Vercel dashboard it is simply stuck on "Auditing..." or says "No report available", so that doesn't help me either. I feel like I have no control or insight into how the integration works. 

Has anybody got it working?

**Comment by timothyis** [comment, marked as answer]
Hi @Jakst. I'm sorry about the experience here. We're working to update this integration to use a much better and stable method of reporting lighthouse results. This should be available this week and I'll make sure to update you here when available.

**Reply by jakst** [reply]
Great, looking forward to it!

**Reply by myylow** [reply]
Hi @timothyis, just to report this issue is still present for me.  Deployments either show as "The deployment is not ready" or get stuck at "Auditing...". We've never seen it add Lighthouse data to a PR yet either. 

This integration has lots of potential to be really useful for us, so I'm very much looking forward it working. 😄
